src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,121 @@
+package wolox.training.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class Book {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+
+    @Column()
+    private String genre;
+
+    @Column(nullable = false)
+    private String author;
+
+    @Column(nullable = false)
+    private String image;
+
+    @Column(nullable = false)
+    private String title;
+
+    @Column(nullable = false)
+    private String subtitle;
+
+    @Column(nullable = false)
+    private String publisher;
+
+    @Column(nullable = false)
+    private String year;
+
+    @Column(nullable = false)
+    private int pages;
+
+    @Column(nullable = false)
+    private String isbn;
+
+    public long getId() {
```

Comment:
You can remplace all the gets and sets with lombok. You have ton only install it in Intelij https://www.baeldung.com/lombok-ide and then add the annotations 
https://kodejava.org/how-do-i-generate-getters-and-setters-with-lombok/

Reply:
When you add Lombok's `@Data` annotation remember to add a restriction in the id setter by adding the following: 

`@Setter(AccessLevel.NONE)`